src/components/button/buttonStyles.tsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
```

Comment:
This file should have .ts extension. Because we don't use JSX syntax here.
Also we don't need import React here.